src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -2591,7 +2591,7 @@ internal void _WriteObjectSkipAllowCheck(object sendToPipeline)
         /// The Cmdlet should generally just allow PipelineStoppedException
         /// to percolate up to the caller of ProcessRecord etc.
         /// </exception>
-        internal void _WriteObjectsSkipAllowCheck(object sendToPipeline)
+        internal void _EnumerateAndWriteObjectSkipAllowCheck(object sendToPipeline)
```

Comment:
```suggestion
        internal void EnumerateAndWriteObjectSkipAllowCheck(object sendToPipeline)
```

Reply:
If I go that route it might be better to just rename half the things in this file, in all honesty... There are a lot of similarly questionable names here.

Reply:
Oh, no. It is better to spend time to create something useful.

Reply:
I'll come back to it when I'm _really_ bored and do a proper style pass over the whole thing. 😉 

Not a big-ticket item, but there's a lot of room for tidying up in this one. 😄